### ISSUE TITLE
Go over README.md, installation instruction, and installs from source

### ISSUE THREAD

**Issue by ndim** [socratools/socranop#50]
As mentioned in https://github.com/socratools/socranop/pull/49, README.md, the installation instructions in README.md, and the instructions for installing from source need a bit of extra attention.

**Comment by lack** [comment]
Do we need instructions on installing from source?  When would users do that?  Which users?

I envision "normal" users using either pip or distro packages.

I envision contributors using whatever we lay out in HACKING.md.

What other use cases and users do you want to support?

**Comment by ndim** [comment]
Agreed: Normal users will install using either distro packages or pip. But then something like the following happens to them:

  * The last release was 10 days ago. We have added a feature 4 days ago. The user wants to use that feature - without waiting until whenever you feel like cutting a new release. *"The official release schedule is sporadic and ad-hoc (aka when I feel like
  it)."*

  * The last release was 10 days ago. The user reported an issue 3 days ago. We think we have fixed that issue yesterday and the user wants to test and then use the fix. Are you going to cut a release every time you fix something? That would be something very different from "sporadic and ad-hoc".
 
Even if your "sporadic and ad-hoc" happens much more frequently than twice a year, there still will be people wanting to try or use the hot new stuff.

I would not go as far as describing a pip install to $HOME from editable local sources at this time, as that would mean writing a completely new set of installations instructions). But at least the pipenv install from editable sources from HACKING.md should be mentioned in README.md and be documented well enough for interested users, even if they are not going to be hacking on the code themselves. That must include at least `git clone` and setting up the pipenv for an initial install, and `git pull -r` to stay up to date.
